packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts

```diff
@@ -0,0 +1,63 @@
+import { container } from '@sapphire/framework';
+import type { Awaited } from '@sapphire/utilities';
+import { ScheduledTaskRedisStrategy } from './strategies/ScheduledTaskRedisStrategy';
+import { ScheduledTaskStore } from './structures/ScheduledTaskStore';
+import type { ScheduledTasksOptions, ScheduledTasksTaskOptions, ScheduledTaskStrategy } from './types';
+
+export class ScheduledTaskHandler {
+	public readonly strategy: ScheduledTaskStrategy;
+
+	public constructor(options: ScheduledTasksOptions | undefined) {
+		this.strategy = options?.strategy ?? new ScheduledTaskRedisStrategy();
+		this.strategy.connect();
```

Comment:
@vladfrangu this will throw `@typescript-eslint/no-floating-promises`. Suggestions?